packages/kit/src/runtime/server/page/load_node.js

```diff
@@ -481,6 +493,17 @@ async function load_shadow_data(route, event, options, prerender) {
 			add_cookies(/** @type {string[]} */ (data.cookies), headers);
 			data.status = status;
 
+			if (body instanceof Error) {
```

Comment:
feels like unnecessary indirection to me — you add function call overhead, a new utility somewhere in the codebase, an import declaration per consuming module... you shave off a few characters at the callsite but make the codebase larger and arguably less easy to read

Reply:
this is one of those cases where you basically _can't_ DRY it out, because of the control flow (i.e. mutating an existing object and conditionally returning it). Or rather you can, but the resulting function is larger than the code you saved by deduplicating. i reckon it's probably not worth it